tools/api_generator/dto_config.yaml

```diff
@@ -0,0 +1,15 @@
+useNewNullableDeterminationStrategy: true
+
+templates:
+  - type: model
+    fileExtension: dart
+    nameSuffix: Data
+    fileNameCase: snakeCase
+    templatePath: templates/retrofit_model.stencil
+    destinationPath: ../../lib/api/data
+  - type: enum
+    fileExtension: dart
+    nameSuffix: Data
+    fileNameCase: snakeCase
+    templatePath: templates/enum_retrofit.stencil
```

Comment:
This template is missing in `templates` folder. It have to be added.

Reply:
He is there:
<img width="722" alt="Screenshot 2024-02-29 at 14 42 36" src="https://github.com/surfstudio/surf-flutter-app-template/assets/63193897/3888a814-4278-4afd-bdf6-636ceed66e9b">

Reply:
Maybe it 's better to name it `retrofit_enum.stencil`?

Reply:
Yep, agree. Let's rename it.